logspout/README.md

```diff
@@ -13,6 +13,7 @@ Tags
 Images in this repository are tagged as follows:
 
 * `0.0.1`: standard [SemVer][3].
+* `0.0.3`: added multiline-logs support.
```

Comment:
I don't think it's needed here, this place is more like description of used tags what are they for.

Reply:
Done